tests/console/nodejs.pm

```diff
@@ -0,0 +1,45 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: NodeJS test to run specific checks availalble on the source code 
+#          about TLS to avoid regressions due to OpenSSL updates.
+#          The test will:
+#          - Check the latest nodejs package available and install it
+#          - Download latest sources of the package from IBS
+#          - Apply patches to the sources
+#          - Run the crypto and tls tests.
+#          - List eventually failed test in the end
+# Maintainer: Michael Grifalconi <mgrifalconi@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils;
```

Comment:
I don't thin you need `version_utils` package

Reply:
removed!